src/features/work-items/get-work-item/feature.ts

```diff
@@ -46,50 +47,7 @@ export async function getWorkItem(
       );
     }
 
-    // Extract project and work item type to get all possible fields
```

Comment:
Hi @saimanish1 👋

This code would need to stay in. It was added because of this bug:

https://github.com/Tiberriver256/mcp-server-azure-devops/issues/160